src/main/java/teammates/ui/controller/InstructorCourseStudentDetailsEditPageAction.java

```diff
@@ -23,6 +25,14 @@ public ActionResult execute() throws EntityDoesNotExistException {
                 instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT);
         
         StudentAttributes student = logic.getStudentForEmail(courseId, studentEmail);
+        
+        if (student == null) {
+            statusToUser.add(new StatusMessage(Const.StatusMessages.STUDENT_NOT_FOUND_EDIT,
+                                               StatusMessageColor.DANGER));
+            isError = true;
+            return createRedirectResult(Const.ActionURIs.INSTRUCTOR_HOME_PAGE);
```

Comment:
That's what the other buttons do.
I suppose it's because the course that the student is in might not exist at all in the first place?